homeassistant/components/dunehd/media_player.py

```diff
@@ -119,57 +135,58 @@ def device_info(self):
         }
 
     @property
-    def volume_level(self):
+    def volume_level(self) -> float:
         """Return the volume level of the media player (0..1)."""
         return int(self._state.get("playback_volume", 0)) / 100
 
     @property
-    def is_volume_muted(self):
+    def is_volume_muted(self) -> bool:
         """Return a boolean if volume is currently muted."""
         return int(self._state.get("playback_mute", 0)) == 1
 
     @property
-    def supported_features(self):
+    def supported_features(self) -> int:
         """Flag media player features that are supported."""
         return DUNEHD_PLAYER_SUPPORT
 
-    def volume_up(self):
+    def volume_up(self) -> None:
         """Volume up media player."""
         self._state = self._player.volume_up()
 
-    def volume_down(self):
+    def volume_down(self) -> None:
         """Volume down media player."""
         self._state = self._player.volume_down()
 
-    def mute_volume(self, mute):
+    def mute_volume(self, mute: Any) -> None:
         """Mute/unmute player volume."""
         self._state = self._player.mute(mute)
 
-    def turn_off(self):
+    def turn_off(self) -> None:
         """Turn off media player."""
         self._media_title = None
         self._state = self._player.turn_off()
 
-    def turn_on(self):
+    def turn_on(self) -> None:
         """Turn off media player."""
         self._state = self._player.turn_on()
 
-    def media_play(self):
+    def media_play(self) -> None:
         """Play media player."""
         self._state = self._player.play()
 
-    def media_pause(self):
+    def media_pause(self) -> None:
         """Pause media player."""
         self._state = self._player.pause()
 
     @property
-    def media_title(self):
+    def media_title(self) -> str | None:
         """Return the current media source."""
         self.__update_title()
         if self._media_title:
```

Comment:
Can `self._media_title` be empty string? Should it return `None` or `""` in this case?

Reply:
If device is playing movie/music from file than `self._state["playback_url"]` always contains file path, for example `/mnt/sda1/movies/movie.mkv` so `self._media_title` will be a non empty string.
https://github.com/home-assistant/core/blob/b539825f829f0f255eb79946871adb8d2ba531b3/homeassistant/components/dunehd/media_player.py#L189-L197